test/cli/cli_install_static_test.go

```diff
@@ -39,19 +39,8 @@ func TestMain(m *testing.M) {
 }
 
 func TestCliInstall(t *testing.T) {
-
-	var (
-		cmd  = "install"
-		args = []string{
-			"--ignore-cluster",
-		}
-	)
-
-	exec := append([]string{cmd}, args...)
-	out, stderr, err := TestHelper.LinkerdRun(exec...)
+	_, err := TestHelper.LinkerdRunOk("install", "--ignore-cluster")
 	if err != nil {
-		testutil.AnnotatedFatalf(t, "'linkerd install' command failed",
-			"'linkerd install' command failed: \n%s\n%s", out, stderr)
+		testutil.AnnotatedFatalf(t, "'linkerd install' command failed", "%s", err)
```

Comment:
The second argument is what is shown in CI as an annotation `"'linkerd install' command failed"`, which should be a generic message with no error details, to be aggregated in the reports.
The third and fourth arguments make the error seen in the CLI. When running outside CI only that will be visible, so it might be a little confusing only seeing the error without the context ("'linkerd install' command failed")

Reply:
would it make sense to prepend the `f.Fatal` message with the annotation message?  That way we can avoid repeating ourselves at the callsite like this.

Reply:
IIRC there are cases where it made sense to make the messages slightly different, but for simplicity's sake I think we can accommodate to your suggestion :+1:

Reply:
AFAICT, there's not a lot of value in including that text again. The new error message includes all of the command line args:
```
fmt.Errorf("command failed: %s\n%s\n%s", strings.Join(arg, " "), err, stderr)
```

Reply:
Ok, command and args should be enough for the CLI message :ok_hand: